app/src/main/java/io/particle/android/sdk/ui/DeviceListFragment.java

```diff
@@ -397,7 +399,7 @@ public String getTextFilter() {
         private Drawable defaultBackground;
         private String textFilter = "";
         private List<ParticleDevice.ParticleDeviceType> typeFilters = list(PHOTON, CORE, ELECTRON,
-                RASPBERRY_PI, P1, RED_BEAR_DUO);
+                RASPBERRY_PI, P1, RED_BEAR_DUO, DIGISTUMP_OAK, BLUZ);
```

Comment:
I think this could just be `Arrays.asList(ParticleDeviceType.values())`, then you don't have to keep specifying them.

This approach could also be used to dynamically build the checkboxes into a proper RecyclerView.